source/_components/light.hyperion.markdown

```diff
@@ -24,12 +26,38 @@ light:
     host: IP_ADDRESS
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The IP address of the device the Hyperion service is running on.
-- **port** (*Optional*): The port used to communicate with the Hyperion service. Defaults to `19444`.
-- **name** (*Optional*): The name of the device used in the frontend.
-- **priority** (*Optional*): The priority of the Hyperion instance. Defaults to `128`.
-- **hdmi_priority** (*Optional*): The priority of the HDMI grabber of this Hyperion instance, note that this priority must be higher than all other priorities used for correct behavior. Defaults to `880`.
-- **default_color** (*Optional*): The color of the light. Defaults to `[255, 255, 255]`.
-- **effect_list** (*Optional*): The list of effects that can be used. Defaults to `['HDMI', 'Cinema brighten lights', 'Cinema dim lights', 'Knight rider', 'Blue mood blobs', 'Cold mood blobs', 'Full color mood blobs', 'Green mood blobs', 'Red mood blobs', 'Warm mood blobs', 'Police Lights Single', 'Police Lights Solid', 'Rainbow mood', 'Rainbow swirl fast', 'Rainbow swirl', 'Random', 'Running dots', 'System Shutdown', 'Snake', 'Sparks Color', 'Sparks', 'Strobe blue', 'Strobe Raspbmc', 'Strobe white', 'Color traces', 'UDP multicast listener', 'UDP listener', 'X-Mas']`.
+{% configuration %}
+  host:
+    description: The IP address of the device the Hyperion service is running on.
+    required: true
+    type: string
+  port:
+    description: The port used to communicate with the Hyperion service.
+    required: false
+    type: int
+    default: 19444
+  name:
+    description: The name of the device used in the frontend.
+    required: false
+    type: string
+  priority:
+    description: The priority of the Hyperion instance.
+    required: false
+    type: int
+    default: 128
+  hdmi_priority:
+    description: The priority of the HDMI grabber of this Hyperion instance, note that this priority must be higher than all other priorities used for correct behavior.
+    required: false
+    type: int
+    default: 880
+  default_color:
+    description: The color of the light.
+    required: false
+    type: string
```

Comment:
This is not a string, but a `list`.
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/light/hyperion.py#L49